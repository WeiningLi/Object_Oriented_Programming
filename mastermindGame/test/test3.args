100001 10 2
