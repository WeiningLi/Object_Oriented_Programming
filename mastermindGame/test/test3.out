Welcome to Mastermind! Please enter your first guess:
0b,7w
1 guesses left. Enter guess:
2b,0w
You lost! The password was: EFBCBADABB
Would you like to play again? (Y/N): 
